client/src/theme/palette.ts

```diff
@@ -10,4 +10,7 @@ export const palette = {
   text: {
```

Comment:
I suggest, it would be better to change the primary main color to 'blueviolet' or something similar